docs/README.md

```diff
@@ -1,34 +1,69 @@
+Tekton Triggers consists of a controller service that runs on your Kubernetes cluster as well as the following Kubernetes Custom Resource Definitions (CRDs) that extend
+the functionality of Tekton Pipelines to support events:
 
-- [`TriggerTemplate`](triggertemplates.md)
-- [`TriggerBinding`](triggerbindings.md)
-- [`EventListener`](eventlisteners.md)
-- [`ClusterTriggerBinding`](clustertriggerbindings.md)
+*  [`EventListener`](https://github.com/tektoncd/triggers/blob/master/docs/eventlisteners.md) - listens for events at a specified port on your Kubernetes cluster.
+   Specifies one or more `Triggers`.
 
+*  [`Trigger`](https://github.com/tektoncd/triggers/blob/master/docs/triggers.md) - specifies what happens when the `EventListener` detects an event. A `Trigger` specifies
+   a `TriggerTemplate`, a `TriggerBinding`, and optionally, an `Interceptor`.
+
+*  [`TriggerTemplate`](https://github.com/tektoncd/triggers/blob/master/docs/triggertemplates.md) - specifies a template for the `TaskRun` or `PipelineRun` you want to
+   instantiate and execute when EventListener detects an event.
+
+*  [`TriggerBinding`](https://github.com/tektoncd/triggers/blob/master/docs/triggerbindings.md) - specifies the fields in the event payload from which you want to extract
+   data as well as the fields in your `TaskRun` or `PipelineRun` to populate with the extracted values. In other words, it *binds* payload fields to `TaskRun` or
+   `PipelineRun` fields.
+
+*  [`ClusterTriggerBinding`](https://github.com/tektoncd/triggers/blob/master/docs/clustertriggerbindings.md) - a cluster-scoped version of the `TriggerBinding`,
+   especially useful for reuse within your cluster.
+
+*  [`Interceptor`](https://github.com/tektoncd/triggers/blob/master/docs/eventlisteners.md#interceptors) - a "catch-all" event processor for a specific platform that
+   runs before the `TriggerBinding` enabling you to perform payload filtering, verification (using a secret), transformation, define and test trigger conditions, and other
+   useful processing. Once the event data passes through an interceptor, it then goes to the `Trigger` before you pass the payload data to the `TriggerBinding`.
+
+
+## What can I do with Triggers?
+
+As an example, you can implement the following CI/CD workflow with Triggers:
+
+1. Triggers listens for a git commit or a git pull request event. When it detects one, it executes a unit test [`Pipeline`](https://github.com/tektoncd/pipeline/blob/master/docs/pipelines.md) on the committed code.
+
+2. Triggers listens for a git push event indicating the test finished successfully. When it detects one, it validates the test's outcome and executes a `Pipeline` that builds the tested code.
+
+3. When the build finishes, Triggers checks the outcome of the build, and if it's successful, executes a [`Task`](https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md) that uploads the build artifacts to the Docker registry of your choice.
```

Comment:
```suggestion
3. When a `PipelineRun` or `TaskRun` finishes, Triggers checks the outcome of the build, and if it's successful, executes a [`Task`](https://github.com/tektoncd/pipeline/blob/master/docs/tasks.md) that uploads the build artifacts to the Docker registry of your choice.
```

Reply:
PipelineRun/TaskRun is more specific than the generic "build"

Reply:
Good point, reworded.